src/server/views/500.html

```diff
@@ -6,7 +6,12 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
-          <p class="govuk-body">Contact your closest consulate.</p>
+          <p class="govuk-body">You can:</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>try your browser back button as your data may be saved</li>
```

Comment:
should the "t" in "try" be capitalised, like how the "C" in "Contact" is?

Reply:
Lowercase is good because it's following the "You can" lead-in

>[**Bulleted lists**](https://design-system.service.gov.uk/styles/lists/#bulleted-lists)
>Introduce bulleted lists with a lead-in line ending in a colon. Start each item with a lowercase letter, and do not use a full stop at the end.